src/BinaryParsers/ElfBinary/Dwarf/DwarfCompilationUnit.cs

```diff
@@ -270,6 +278,7 @@ private void ReadData(IDwarfBinary dwarfBinary, DwarfMemoryReader debugData, Dwa
                             break;
 
                         default:
+                            Console.WriteLine($"No case for format {format}");
```

Comment:
>Console [](http://example.com/codeflow?start=28&length=7)

This debug output was helpful as it reports out on unknown format types. We should remove it though, it shouldn't be in production.